modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js

```diff
@@ -161,6 +162,42 @@ class ContributorBuilder extends React.Component {
 
 							<ClayLayout.ContainerFluid>
 								<div className="content-wrapper">
+									{Liferay.FeatureFlags['LPS-166954'] && (
+										<ClayPanel
+											collapsable
+											defaultExpanded={true}
+											displayTitle={
+												<h2 className="mb-0 size- text-dark">
```

Comment:
The class `size-` is not correct.  Also, maybe it is good to use the ClayPanel.Header component.